src/SwedbankPay.Sdk.Infrastructure/SwedbankPayClient.cs

```diff
@@ -1,28 +1,34 @@
-﻿using System;
+﻿using SwedbankPay.Sdk.Consumers;
+using SwedbankPay.Sdk.PaymentInstruments;
+using SwedbankPay.Sdk.PaymentOrders;
+using System;
 using System.Net;
 using System.Net.Http;
-using SwedbankPay.Sdk.Consumers;
-using SwedbankPay.Sdk.Payments;
-using SwedbankPay.Sdk.PaymentOrders;
 
 namespace SwedbankPay.Sdk
 {
     public class SwedbankPayClient : ISwedbankPayClient
     {
         private readonly HttpClient httpClient;
 
-        public SwedbankPayClient(HttpClient httpClient, IPaymentOrdersResource paymentOrders, IConsumersResource consumers, IPaymentsResource payments)
+        public SwedbankPayClient(HttpClient injectedHttpClient, IPaymentOrdersResource paymentOrders, IConsumersResource consumers, IPaymentInstrumentsResource payments)
```

Comment:
Why `injectedHttpClient`?

Reply:
This was a result of the confusion of the `this.` spectacle.